package.json

```diff
@@ -39,6 +39,6 @@
     "lodash": "^4.17.5",
     "loopback-swagger": "^5.0.0",
     "strong-globalize": "^4.1.1",
-    "swagger-ui": "^2.2.5"
+    "swagger-ui": "^2.2.10"
```

Comment:
The pull request mentions upgrade to `swagger-ui@3`, but this change is upgrading only to `2.2.10`. Since we are already using `^` version specifier, such change is not necessary at all - `npm` is going to install `2.2.10` anyway.

Reply:
The reason I submit this PR is to get rid of the vulnerabilities.  I thought the warnings go away when I change the version number.  Let me try that again.  thanks!